README.rst

```diff
@@ -135,6 +135,12 @@ You will be asked for these fields:
       - Use ``"no"`` for no hosting (various links will disappear). You can also use ``"gitlab"`` and such but various
         things will be broken (like Travis configuration).
 
+    * - ``ci_https_proxy``
```

Comment:
Could be "gitlab_ci_http_proxy" or similar, unless we'd support other on-premise ci in the future ... hmmm.

Reply:
Actually nevermind.